### PR TITLE
Use PyQt5 as default API

### PR DESCRIPTION
**Pull request by flying-sheep**
i think i did this once.

you stated then that it wasn’t ready, but that’s been some time ago and there are now PyQt5 travis tests.

also 3.0 should really ship like that.

### CODE REVIEW

**Comment by goanpeca** [comment]
@flying-sheep, until there is an official qt5 package in anaconda, I do not think we are doing this.

**Comment by flying-sheep** [comment]
if the API can’t be found, it will fall back to PyQt4, so this change does nothing for people without PyQt5.

the only thing it does is preferring to use PyQt5 if available, which is something that all Qt software should do now.

**Comment by flying-sheep** [comment]
oh it actually doesn’t, i’ll have to reorder things!

**Comment by ccordoba12** [comment]
@goanpeca, I think beta3 is a good point to move to PyQt5 :-)

@flying-sheep, tests for Qt5 are failing in Travis. Please solve the failure.

**Comment by ccordoba12** [comment]
@flying-sheep, I think I fixed the Travis error in master, so please rebase or merge with master to trigger a new build.

However, I think your change breaks Spyder when both PyQt4 and PyQt5 are installed. Please check that in your side.

**Comment by flying-sheep** [comment]
ok, will do

**Comment by ccordoba12** [comment]
Now this breaks **all** our tests in Travis. I really don't know what happens...

**Comment by flying-sheep** [comment]
the non-conda builds break on

`python spyderlib/utils/environ.py`

which hangs indefinitely, because it opens a dialog. why does the travis build call that?

**Comment by flying-sheep** [comment]
ok, the tests work flawlessly here.

unfortunately i neither have any idea what goes wrong with the CI tests.

File: spyderlib/qt/__init__.py [hunk 1]
```diff
@@ -9,12 +9,23 @@
...
+PYQT5 = True
+
+if API == 'pyqt5':
+    try:
+        from PyQt5.QtCore import PYQT_VERSION_STR as __version__
+        from PyQt5 import uic  # analysis:ignore
+    except ImportError:
+        API = os.environ['QT_API'] = 'pyqt'
+        API_NAME = 'PyQt4'
+
```
Comment (ccordoba12) [inline spyderlib/qt/__init__.py]: I think we should set `PYQT5 = False` here.

Comment (ccordoba12) [inline spyderlib/qt/__init__.py]: IMHO, it's easier to read and understand.

**Comment by ccordoba12** [comment]
Except for the comment I've just made, I think this ready.

**Comment by flying-sheep** [comment]
fixed

**Comment by ccordoba12** [comment]
Ok, I think this is ready! Merging :-)

**Comment by flying-sheep** [comment]
great :smile:

**Pull request merged**